.github/workflows/test.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Test using kind and chart-testing tool"
+on: [pull_request, push]
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetesVersion: ["v1.13.12", "v1.17.0"]
```

Comment:
I'm curious, why these two specific versions?

Reply:
copy&paste from my current setup - the values are based on latest available release, and the current version of our cluster 1.13.x)